### PR TITLE
Fix NPM static analysis

### PR DESCRIPTION
**Pull request by mattnorris**
It looks like packages may have been locked on the last run. This copies the config into the home directory to ensure the project config is maintained before adding the new token.

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit 1aed8401d2c9552e809fa1cfbf1c15bf1309fbb6

https://deploy-preview-749--hopeful-hamilton-4090b9.netlify.com

File: Jenkinsfile [hunk 1]
```diff
@@ -207,7 +207,10 @@ ansiColor('xterm') {
                 string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
               ]) {
                 try {
-                  sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > $HOME/.npmrc'
+                  // Copy & update config file
+                  sh 'cp .npmrc $HOME/.npmrc'
```
Comment (adamweeks) [inline Jenkinsfile]: Did we want to backup the existing npmrc in the home directory if it exists since other teams might be relying on it?

Comment (mattnorris) [reply Jenkinsfile]: The web team confirmed that they weren't. I'd rather have each team be responsible for setting up and tearing down the environments properly than perpetuate dependence on files that may not be there.

Comment (adamweeks) [reply Jenkinsfile]: Sound good to me. LGTM!

**Pull request merged**